src/server.jl

```diff
@@ -799,6 +845,33 @@ function process_cell_source(source::AbstractString, cell_options::Dict = Dict()
     end
 end
 
+function strip_cell_options(source::AbstractString)
+    lines = split(source, "\n")
```

Comment:
Elsewhere for line splitting we've used `eachline(IOBuffer(source); keep = true)`, can this be adjusted to be consistent with those places?